packages-common/piw-utils/src/components.tsx

```diff
@@ -7,7 +7,7 @@ export interface AlertProps {
     bootstrapStyle: "default" | "primary" | "success" | "info" | "warning" | "danger";
 }
 
-export const Alert = ({ className, bootstrapStyle, children }: AlertProps): ReactNode =>
+export const Alert = ({ className, bootstrapStyle, children }: AlertProps): JSX.Element | null =>
```

Comment:
Do we still need `Alert`?

Reply:
Not for this widget but we can reuse in web widgets in general.